tests/fixtures/basics.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2020 RERO
+# Copyright (C) 2020 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Common pytest fixtures for rero-ils."""
+from datetime import datetime, timedelta, timezone
+
+import pytest
+
+
+@pytest.fixture(scope="module")
+def yesterday():
+    """Get yesterday timestamp."""
+    return datetime.now(timezone.utc) - timedelta(days=1)
+
+
+@pytest.fixture(scope="module")
+def tomorrow():
+    """Get tomorrow timestamp."""
+    return datetime.now(timezone.utc) + timedelta(days=1)
```

Comment:
Shall we use `datetime.utcnow()` everywhere?

Reply:
This notation is a legacy from old unit test. See here : https://github.com/rero/rero-ils/blob/dev/tests/ui/holdings/test_serial_claims.py#L61